src/Microsoft.Framework.Runtime.Hosting/Dependencies/DependencyManager.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Runtime.Internal;
+using NuGet.DependencyResolver;
+using NuGet.Frameworks;
+using NuGet.LibraryModel;
+using NuGet.Versioning;
+
+namespace Microsoft.Framework.Runtime.Dependencies
+{
+    /// <summary>
+    /// Represents the dependency graph for a project
+    /// </summary>
+    public class DependencyManager
+    {
+        private static readonly ILogger Log = RuntimeLogging.Logger<DependencyManager>();
+
+        private readonly GraphNode<Library> _graph;
```

Comment:
They are an artifact of an earlier bit of code I wrote to look things up. I'll remove these for now.